spec.md

```diff
@@ -549,6 +549,28 @@ Analog<32> ; 32-bit analog type
 Analog     ; analog type with inferred width
 ```
 
+### Enumeration Types
```

Comment:
Yea, this is a hard one.  they are pretty ground-like in that they don't get decomposed by lower types, but they are more aggregate like.  I've moved them to aggregate, though this may require some other changes.